app/forms/date_of_birth_form.rb

```diff
@@ -18,17 +18,63 @@ def date_of_birth
     @date_of_birth ||= trn_request&.date_of_birth
   end
 
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
Here I disagreed with Rubocop that breaking up this method would make it more readable :thinking: I think it reads really well like it is.

Reply:
Yeah, I'm fine with this too. I don't think it is complex, just a long method.